sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc

```diff
@@ -0,0 +1,83 @@
+[id="sandboxed-containers-4-9-release-notes"]
+= {sandboxed-containers-first} {product-version} release notes
+include::modules/common-attributes.adoc[]
+:context: sandboxed-containers-release-notes
+
+toc::[]
+
+[id="ocp-4-9-about-this-release"]
+== About this release
+
+These release notes track the development of {sandboxed-containers-first} in Red Hat {product-title}.
+
+This product is currently in Technology Preview. {sandboxed-containers-first} is not intended for production use. For more information, see the Red Hat Customer Portal link:https://access.redhat.com/support/offerings/techpreview[support scope] for features in Technology Preview.
+
+[id="sandboxed-containers-4-9-new-features-and-enhancements"]
+== New features and enhancements
+
+[id="fips-compatability-rn"]
+=== FIPS compatibility
+
+FIPS mode is now automatically enabled for {sandboxed-containers-first}. {sandboxed-containers-first} deployed on an {product-title} cluster installed in FIPS mode will not taint the cluster's FIPS support. For more information, see xref:../sandboxed_containers/understanding-sandboxed-containers.adoc#security-compliance-nist_understanding-sandboxed-containers[Understanding compliance and risk management].
+
+[id="collect-resources-with-must-gather-rn"]
+=== Collect resources with must-gather
+
+The {sandboxed-containers-operator} now includes a must-gather image, allowing you to collect custom resources and log files specific to this operator and the underlying runtime components for diagnostic purposes. For more information, see xref:../sandboxed_containers/troubleshooting-sandboxed-containers.adoc#troubleshooting-sandboxed-containers[Collecting {sandboxed-containers-first} data for Red Hat Support].
+
+[id="sandboxed-containers-disconnected-environments-rn"]
+=== Disconnected environments
+
+You can now install the Sandboxed container operator in a disconnected environment. For more information, see the xref:../sandboxed_containers/deploying-sandboxed-container-workloads.adoc#additional-resources[Additional resources for Deploying {sandboxed-containers-first} workloads].
```

Comment:
For this Operator title, I'm not sure what we're recommending for capitalization. But we'll need to capitalize "Operator" everywhere. So either:
- Sandboxed containers Operator
- Sandboxed Containers Operator

Reply:
Thanks - yeah, it's the first bullet. My bad. I keep forgetting because of the variable. I'll update it rn.